src/error.rs

```diff
@@ -127,6 +130,12 @@ impl ApiError {
         true
     }
 
+    pub fn on_response(&self, state: &Data<ServerState>) {
```

Comment:
A post-merge nit: This should be `&ServerState` because `Data<T>` is an actix-specific type just used for extracting stuff from app data, and dereferences to `&T`.

Reply:
sigh. yeah, good point. I'll do a fixup PR on this monday.